pom.xml

```diff
@@ -195,6 +195,20 @@
     </pluginRepositories>
 
     <build>
+        <resources>
+            <resource>
+                <directory>${project.basedir}</directory>
```

Comment:
This will break checkstyle when running `mvn release` , need to update checkstyle section to only include resources folder.